websites/W/wumpus.store/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "998532462755979275",
+	}),
+	browsingUnix = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/KaW8pr2.png",
+			details: "Viewing Home Page",
+			startTimestamp: browsingUnix,
+		},
+		{ pathname, origin } = document.location;
+
+	let title: HTMLElement, logo: HTMLElement;
+
+	switch (pathname) {
+		case "/bot/add": {
+			presenceData.details = "Adding a new bot";
+			break;
+		}
+		case "/search": {
+			presenceData.details = "Searching for a new bot";
+			break;
+		}
+		case "/": {
+			presenceData.details = "Viewing Home Page";
+			break;
+		}
+	}
+
+	if (
+		pathname.startsWith("/bot/") &&
+		document.querySelector('b[class*="chakra-text"]')
+	) {
+		title = document.querySelector('b[class*="chakra-text"]');
+		logo =
+			document
+				.querySelectorAll('[class*="chakra-container"]')?.[1]
+				?.querySelector('[class*="chakra-avatar__img"]') ??
+			document.querySelector('[class*="chakra-avatar__img"]');
+		presenceData.details = "Viewing Bot:";
+		presenceData.state = title.textContent;
+		presenceData.smallImageKey = logo.getAttribute("src");
+
+		presenceData.buttons = [
+			{
+				label: "Visit Bot Page",
+				url: `${origin}/bot/${pathname.split("/")[2]}/`,
+			},
+			{
+				label: `Vote for ${title.textContent}`,
```

Comment:
@Bas950 Is this allowed, it wasn’t before so not sure. (its not in the [guidelines ](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#revision-3))